code/src/index.css

```diff
@@ -1,10 +1,16 @@
 body {
```

Comment:
very nice how you've split the css up into separate files, makes it so easy and digestible to read!

Reply:
Agree, the structure of the css and the classnames makes it very clear, nice!